test/conversions.jl

```diff
@@ -351,9 +352,10 @@ end
         @test convert_arguments(Heatmap, r1, i2, m3) == (o1, o2, o3)
         @test convert_arguments(Heatmap, v1, r2, m3) == (o1, o2, o3)
         @test convert_arguments(Heatmap, 0:10, v2, m3) == (collect(0f0:10f0), o2, o3)
-        @test_throws ErrorException convert_arguments(Heatmap, m1, m2, m3)
-        @test_throws ErrorException convert_arguments(Heatmap, m1, m2)
-
+        # TODO, this throws ERROR: MethodError: no method matching adjust_axes(::CellGrid, ::Matrix{Int64}, ::Matrix{Int64}, ::Matrix{Float64})
+        # Is this what we want to test for?
+        @test_throws MethodError convert_arguments(Heatmap, m1, m2, m3) === (m1, m2, m3)
+        @test convert_arguments(Heatmap, m1, m2) === (m1, m2)
```

Comment:
Something like `[1 2; 1.5 2.5]` doesn't work for heatmaps since the cells are rectangular. So you can't use the extra freedom a Matrix input gives you. I think that's what the missing conversion/test is implying.